.kokoro/build.sh

```diff
@@ -33,9 +27,11 @@ export PYTHONUNBUFFERED=1
 # Debug: show build environment
 env | grep KOKORO
 
-# Install pip
-python3 -m pip install --upgrade --quiet pip
-python3 -m pip --version
+# Setup service account credentials.
+export GOOGLE_APPLICATION_CREDENTIALS=${KOKORO_GFILE_DIR}/service-account.json
```

Comment:
This is unnecessary since we're using the credentials associates with the Compute Engine Instance.